apps/webapp/src/components/header/header.tsx

```diff
@@ -5,29 +5,44 @@ import Notifications from './notifications.tsx';
 import { TabletNavMenu } from './tablet-nav-menu.tsx';
 import { LayoutLoaderResult } from '../layout.tsx';
 import { NetworksPopover } from '@penumbra-zone/ui';
+import { MobileNavMenu } from './mobile-nav-menu.tsx';
 
 export const Header = () => {
   const result = useLoaderData() as LayoutLoaderResult;
 
   return (
-    <header className='flex h-[82px] w-full items-center justify-between px-12'>
-      <img
-        src='/penumbra-logo.svg'
-        width={234}
-        height={234}
-        alt='Penumbra logo'
-        className='absolute left-[-100px] top-[-140px] rotate-[320deg]'
-      />
-      <Link to={PagePath.INDEX}>
-        <img src='/logo.svg' alt='Penumbra logo' className='relative z-10 h-4 w-[171px]' />
-      </Link>
+    <header className='z-10 flex h-[82px] w-full flex-col items-center justify-between px-6 md:flex-row md:px-12'>
+      <div className='mb-[46px] md:mb-0'>
+        <img
+          src='/penumbra-logo.svg'
+          width={234}
+          height={234}
+          alt='Penumbra logo'
+          className='absolute inset-x-0 top-[-85px] mx-auto h-[126px] w-[126px] rotate-[320deg] md:left-[-100px] md:top-[-140px] md:mx-0 md:h-[234px] md:w-[234px]'
+        />
```

Comment:
The logo looks very small and the rays are too small.

<img width="315" alt="Screenshot 2023-11-13 at 3 32 17 PM" src="https://github.com/penumbra-zone/web/assets/16624263/a063a453-a727-433f-930f-240792725606">

Should be closer to this size:
<img width="603" alt="Screenshot 2023-11-13 at 3 33 44 PM" src="https://github.com/penumbra-zone/web/assets/16624263/5a9d0ebb-8340-4c55-bd88-3040e0d8469b">